remote-config.jsonc

```
@@ -93,6 +93,36 @@
                 },
 				{
                     "message": "MacStadium supports DDEV with an M1 Mac Mini that runs macOS tests"
+                },
+                {
+                    "message": "You can turn off the cell borders in `ddev describe` and `ddev list` with `ddev config global --simple-formatting`"
+                },
+                {
+                    "message": "You can use a more lively `ddev list` and `ddev describe` output with `ddev config global --table-style=bright`. Try it out!"
+                },
+                {
+                    "message": "DDEV's fiscal entity is now the DDEV Foundation. http://ddev.com/foundation. We have a real focus on the long-term sustainability of the project, not just depending on Randy."
+                },
+                {
+                    "message": "Tag1 Consulting has become a major sponsor of DDEV with $1K/month! Stop by and thank them, https://tag1.com."
+                },
+                {
+                    "message": "DDEV now has PHP 8.3 in latest releases. We hope to have all extensions by the time of 8.3 release."
+                },
+                {
+                    "message": "Lots of database browsers work with DDEV, including TablePlus, Sequel Pro, and DBeaver. https://ddev.readthedocs.io/en/latest/users/usage/database-management/#database-guis"
+                },
+                {
+                    "message": "View officially supported add-ons with `ddev get --list` or ALL of them with `ddev get --list --all`"
+                },
+                {
+                    "message": "vscode users: Have you tried the new DDEV vscode extension? https://marketplace.visualstudio.com/items?itemName=biati.ddev-manager"
+                },
+                {
+                    "message": "PhpStorm users: You'll love the DDEV Integration Plugin for PhpStorm, https://plugins.jetbrains.com/plugin/18813-ddev-integration"
+                },
+                {
+                    "message": "Did you know you can set environment variables with the .ddev/.env file?"
```

If commands are in backticks (like the docs), this file reference should be too:

```suggestion
                    "message": "Did you know you can set environment variables with the `.ddev/.env` file?"
```